UnityProject/Assets/Scripts/Managers/ChatRelay.cs

```diff
@@ -38,7 +38,7 @@ public override void OnStartClient()
 			{ChatChannel.Common, "#008000"},
 			{ChatChannel.Engineering, "#fb5613"},
 			{ChatChannel.Examine, "black"},
-			{ChatChannel.Local, "#999999"},
+			{ChatChannel.Local, "black"},
```

Comment:
Shoot the person who started using "black" here btw... will change it to proper hex one day...